assets/icons/return-delivery-home.svg

```diff
@@ -0,0 +1,4 @@
+<svg id="b08c42c4-af94-4465-ba97-f5cffd70f242" data-name="Export here" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 200">
+  <title>Master (Dont Touch!)</title>
```

Comment:
While we have been very poor with using correct titles, try and get them in when we can